modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/communicators/aah_async_comm.h

```diff
@@ -24,42 +22,44 @@ class FLUDS;
 class AAH_ASynchronousCommunicator : public AsynchronousCommunicator
 {
 private:
-  const size_t num_groups_;
-  const size_t num_angles_;
-
-  bool done_sending;
-  bool data_initialized;
-  bool upstream_data_initialized;
+  int num_groups_;
+  int num_angles_;
+  int max_num_messages_;
+  int max_mpi_message_size_;
+  bool done_sending_;
+  bool data_initialized_;
+  bool upstream_data_initialized_;
 
-  u_ll_int EAGER_LIMIT = 32000;
+  std::vector<int> prelocI_message_count_;
+  std::vector<int> deplocI_message_count_;
+  std::vector<int> delayed_prelocI_message_count_;
 
-  std::vector<int> prelocI_message_count;
-  std::vector<int> deplocI_message_count;
-  std::vector<int> delayed_prelocI_message_count;
+  std::vector<std::vector<size_t>> prelocI_message_size_;
+  std::vector<std::vector<size_t>> deplocI_message_size_;
+  std::vector<std::vector<size_t>> delayed_prelocI_message_size_;
 
-  std::vector<std::vector<u_ll_int>> prelocI_message_size;
-  std::vector<std::vector<u_ll_int>> deplocI_message_size;
-  std::vector<std::vector<u_ll_int>> delayed_prelocI_message_size;
+  std::vector<std::vector<size_t>> prelocI_message_blockpos_;
+  std::vector<std::vector<size_t>> deplocI_message_blockpos_;
+  std::vector<std::vector<size_t>> delayed_prelocI_message_blockpos_;
 
-  std::vector<std::vector<u_ll_int>> prelocI_message_blockpos;
-  std::vector<std::vector<u_ll_int>> deplocI_message_blockpos;
-  std::vector<std::vector<u_ll_int>> delayed_prelocI_message_blockpos;
+  std::vector<std::vector<bool>> prelocI_message_received_;
+  std::vector<std::vector<bool>> delayed_prelocI_message_received_;
 
-  std::vector<std::vector<bool>> prelocI_message_received;
-  std::vector<std::vector<bool>> delayed_prelocI_message_received;
-
-  std::vector<std::vector<mpi::Request>> deplocI_message_request;
+  std::vector<std::vector<mpi::Request>> deplocI_message_request_;
 
 public:
-  int max_num_mess;
-
   AAH_ASynchronousCommunicator(FLUDS& fluds,
                                size_t num_groups,
                                size_t num_angles,
-                               int sweep_eager_limit,
+                               int max_mpi_message_size,
                                const MPICommunicatorSet& in_comm_set);
+
+  int GetMaxNumMessages() const { return max_num_messages_; }
+
+  void SetMaxNumMessages(int count) { max_num_messages_ = count; }
+
   /**
-   * Returns the private flag done_sending.
+   * Returns the private flag done_sending_.
```

Comment:
This seems like a product of search-and-replace, but it would deserve better wording